cylc/flow/config.py

```diff
@@ -2265,6 +2266,17 @@ def load_graph(self):
             self.workflow_polling_tasks.update(
                 parser.workflow_state_polling_tasks)
             self._proc_triggers(parser, seq, task_triggers)
+            self.check_outputs(
```

Comment:
Can you add a comment that this is just checking "terminals" and non-terminals are checked elsewhere.